moveit_core/collision_distance_field/src/collision_env_distance_field.cpp

```diff
@@ -35,54 +35,61 @@
 /* Author: E. Gil Jones */
 
 #include <moveit/robot_model/robot_model.h>
-#include <moveit/collision_distance_field/collision_robot_distance_field.h>
-#include <moveit/collision_distance_field/collision_common_distance_field.h>
-#include <moveit/distance_field/propagation_distance_field.h>
 #include <ros/console.h>
 #include <ros/assert.h>
 #include <tf2_eigen/tf2_eigen.h>
+#include <moveit/collision_distance_field/collision_env_distance_field.h>
+#include <moveit/collision_distance_field/collision_common_distance_field.h>
+#include <moveit/distance_field/propagation_distance_field.h>
+#include <moveit/collision_distance_field/collision_detector_allocator_distance_field.h>
+#include <boost/bind.hpp>
+#include <memory>
+#include <utility>
```

Comment:
Are there any unit tests for this collision checker? Maybe add some to make sure there are no breakages, making sure they pass before and after this PR?

Reply:
Yes there is in [moveit_core/collision_distance_field/test/test_collision_distance_field.cpp](https://github.com/ros-planning/moveit/pull/1584/files?file-filters%5B%5D=.cpp#diff-9a16e0b8917fb63dce1d48d83243106b) and all pass.